src/crypto/aws_kms.ts

```diff
@@ -0,0 +1,52 @@
+import * as AWS from "aws-sdk"
+import * as winston from "winston"
+import {CryptoBase} from "./crypto_base"
+
+export class AwsKms extends CryptoBase {
```

Comment:
Is there a place that you check to ensure that everything required to use AWS KMS is set? Something on the lines of  a check to ensure that the Environment variable pointing to the CMK is set here, and more if required. the check failure can (must) provide useful actionable error message.